{"name":"Setting up validation rules: Setup.thn__ByPass__c.thn__ByPassVR__c == false and User.thn__ByPassVR__c == false ","status":"broken","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"JSONObject[\"result\"] not found.","trace":"org.json.JSONException: JSONObject[\"result\"] not found.\n\tat org.json.JSONObject.get(JSONObject.java:572)\n\tat org.json.JSONObject.getJSONObject(JSONObject.java:782)\n\tat pageObject.JsonParser2.getFieldValueSoql(JsonParser2.java:63)\n\tat tests.ValidationRules.ValidationRule1.settingUpValidationRules(ValidationRule1.java:59)\n\tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\n\tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:78)\n\tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\n\tat java.base/java.lang.reflect.Method.invoke(Method.java:567)\n\tat org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)\n\tat org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:584)\n\tat org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:172)\n\tat org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)\n\tat org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:804)\n\tat org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:145)\n\tat org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)\n\tat org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)\n\tat java.base/java.util.ArrayList.forEach(ArrayList.java:1511)\n\tat org.testng.TestRunner.privateRun(TestRunner.java:770)\n\tat org.testng.TestRunner.run(TestRunner.java:591)\n\tat org.testng.SuiteRunner.runTest(SuiteRunner.java:402)\n\tat org.testng.SuiteRunner.runSequentially(SuiteRunner.java:396)\n\tat org.testng.SuiteRunner.privateRun(SuiteRunner.java:355)\n\tat org.testng.SuiteRunner.run(SuiteRunner.java:304)\n\tat org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)\n\tat org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)\n\tat org.testng.TestNG.runSuitesSequentially(TestNG.java:1180)\n\tat org.testng.TestNG.runSuitesLocally(TestNG.java:1102)\n\tat org.testng.TestNG.runSuites(TestNG.java:1032)\n\tat org.testng.TestNG.run(TestNG.java:1000)\n\tat com.intellij.rt.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:66)\n\tat com.intellij.rt.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:110)\n\tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\n\tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:78)\n\tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\n\tat java.base/java.lang.reflect.Method.invoke(Method.java:567)\n\tat com.intellij.rt.execution.application.AppMainV2.main(AppMainV2.java:131)\n"},"stage":"finished","steps":[{"name":"Run SFDX command","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[{"name":"cmd","value":"[/home/minsk-sc/sfdx/bin/sfdx, force:auth:jwt:grant, --clientid, 3MVG9KlmwBKoC7U13f3dBKdwrpeVJmU4xkpDgywAH_NMf8B0S.B2iOHg_ajPoBs5BQqeH7Wkz6aLRrgZF1NaR, --jwtkeyfile, /home/user/jdoe/JWT/server.key, --username, test-sy9dufviyszi@example.com, --instanceurl, https://test.salesforce.com]"}],"start":1633078326672,"stop":1633078329915},{"name":"Run SFDX command","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[{"name":"cmd","value":"[/home/minsk-sc/sfdx/bin/sfdx, force:data:record:update, -s, User, -w, Name='User User', -v, thn__ByPassVR__c=false, -u, test-sy9dufviyszi@example.com, --json]"}],"start":1633078329916,"stop":1633078330367},{"name":"Run SFDX command","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[{"name":"cmd","value":"[/home/minsk-sc/sfdx/bin/sfdx, force:data:soql:query, -q, SELECT Id FROM thn__bypass__c, -u, test-sy9dufviyszi@example.com, --json]"}],"start":1633078330367,"stop":1633078330780},{"name":"Get value from SOQL","status":"broken","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"JSONObject[\"result\"] not found.","trace":"org.json.JSONException: JSONObject[\"result\"] not found.\n\tat org.json.JSONObject.get(JSONObject.java:572)\n\tat org.json.JSONObject.getJSONObject(JSONObject.java:782)\n\tat pageObject.JsonParser2.getFieldValueSoql(JsonParser2.java:63)\n\tat tests.ValidationRules.ValidationRule1.settingUpValidationRules(ValidationRule1.java:59)\n\tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\n\tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:78)\n\tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\n\tat java.base/java.lang.reflect.Method.invoke(Method.java:567)\n\tat org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)\n\tat org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:584)\n\tat org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:172)\n\tat org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)\n\tat org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:804)\n\tat org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:145)\n\tat org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)\n\tat org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)\n\tat java.base/java.util.ArrayList.forEach(ArrayList.java:1511)\n\tat org.testng.TestRunner.privateRun(TestRunner.java:770)\n\tat org.testng.TestRunner.run(TestRunner.java:591)\n\tat org.testng.SuiteRunner.runTest(SuiteRunner.java:402)\n\tat org.testng.SuiteRunner.runSequentially(SuiteRunner.java:396)\n\tat org.testng.SuiteRunner.privateRun(SuiteRunner.java:355)\n\tat org.testng.SuiteRunner.run(SuiteRunner.java:304)\n\tat org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)\n\tat org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)\n\tat org.testng.TestNG.runSuitesSequentially(TestNG.java:1180)\n\tat org.testng.TestNG.runSuitesLocally(TestNG.java:1102)\n\tat org.testng.TestNG.runSuites(TestNG.java:1032)\n\tat org.testng.TestNG.run(TestNG.java:1000)\n\tat com.intellij.rt.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:66)\n\tat com.intellij.rt.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:110)\n\tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\n\tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:78)\n\tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\n\tat java.base/java.lang.reflect.Method.invoke(Method.java:567)\n\tat com.intellij.rt.execution.application.AppMainV2.main(AppMainV2.java:131)\n"},"stage":"finished","steps":[],"attachments":[],"parameters":[{"name":"json","value":"{\n  \"status\": 1,\n  \"name\": \"NamedOrgNotFound\",\n  \"message\": \"No AuthInfo found for name test-sy9dufviyszi@example.com\",\n  \"exitCode\": 1,\n  \"commandName\": \"DataSoqlQueryCommand\",\n  \"stack\": \"NamedOrgNotFound: No AuthInfo found for name test-sy9dufviyszi@example.com\\n    at Function.create (/home/minsk-sc/sfdx/node_modules/@salesforce/core/lib/sfdxError.js:160:16)\\n    at AuthInfo.loadAuthFromConfig (/home/minsk-sc/sfdx/node_modules/@salesforce/core/lib/authInfo.js:716:49)\\n    at async AuthInfo.initAuthOptions (/home/minsk-sc/sfdx/node_modules/@salesforce/core/lib/authInfo.js:690:26)\\n    at async AuthInfo.init (/home/minsk-sc/sfdx/node_modules/@salesforce/core/lib/authInfo.js:619:13)\\n    at async Function.create (/home/minsk-sc/sfdx/node_modules/@salesforce/kit/lib/creatable.js:31:9)\\n    at async Org.init (/home/minsk-sc/sfdx/node_modules/@salesforce/core/lib/org.js:403:27)\\n    at async Function.create (/home/minsk-sc/sfdx/node_modules/@salesforce/kit/lib/creatable.js:31:9)\\n    at async DataSoqlQueryCommand.assignOrg (/home/minsk-sc/sfdx/node_modules/@salesforce/command/lib/sfdxCommand.js:109:24)\\n    at async DataSoqlQueryCommand.init (/home/minsk-sc/sfdx/node_modules/@salesforce/command/lib/sfdxCommand.js:231:13)\\n    at async DataSoqlQueryCommand._run (/home/minsk-sc/sfdx/node_modules/@salesforce/command/lib/sfdxCommand.js:80:13)\",\n  \"warnings\": []\n}\n"},{"name":"keys","value":"[Id]"}],"start":1633078330780,"stop":1633078330794}],"attachments":[{"name":"Page screenshot","source":"f949f467-4660-43f4-a39d-e1891de358d4-attachment","type":"image/png"},{"name":"settingUpValidationRules failed and screenshot taken!","source":"2b9821d7-f83a-47ca-bca8-86ab5239e931-attachment","type":"text/plain"},{"name":"Browser Log","source":"e5093d0c-6180-4de2-addb-384082be6700-attachment","type":"text/plain"}],"parameters":[],"start":1633078326661,"stop":1633078331068,"uuid":"5cb32e75-fe16-4f10-8b3e-0b27c7f482e8","historyId":"ad8e1031e4795d817293220bd38863b4","fullName":"tests.ValidationRules.ValidationRule1.settingUpValidationRules","labels":[{"name":"package","value":"tests.ValidationRules.ValidationRule1"},{"name":"testClass","value":"tests.ValidationRules.ValidationRule1"},{"name":"testMethod","value":"settingUpValidationRules"},{"name":"parentSuite","value":"Default Suite"},{"name":"suite","value":"SCNew"},{"name":"subSuite","value":"tests.ValidationRules.ValidationRule1"},{"name":"host","value":"minsksc-desktop"},{"name":"thread","value":"14282@minsksc-desktop.main(1)"},{"name":"framework","value":"testng"},{"name":"language","value":"java"},{"name":"story","value":"THY-510: Validation rule updated"},{"name":"severity","value":"normal"}],"links":[]}